### PR TITLE
Added text alignment functions

### LINKED ISSUE #823: utils.center_text() is not ansi aware
The `utils.center_text()` function will produce incorrect results if the text to be centered contains ANSI escape sequences. Advise modifying the method to calculate the padding using `ansi.ansi_safe_wcswidth()`.

### PR DESCRIPTION
**Pull request by kmvanbrunt**
Fixes #823

### CODE REVIEW

File: cmd2/utils.py [hunk 1]
```diff
@@ -644,3 +629,154 @@ def basic_complete(text: str, line: str, begidx: int, endidx: int, match_against
...
+
+
+class TextAlignment(Enum):
+    LEFT = 1
+    CENTER = 2
+    RIGHT = 3
+
+
+def align_text(text: str, *, fill_char: str = ' ', width: Optional[int] = None, tab_width: int = 4,
+               alignment: TextAlignment) -> str:
```
Comment (tleonhardt) [inline cmd2/utils.py]: `alignment` appears to be a required keyword-only argument.  I would recommend doing one of the following:
* Make it a positional argument
* Move it to be the 1st keyword-only argument
* Give it a default value

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
-## 0.9.22 (TBD, 2019)
+## 0.9.22 (December 10, 2019)
 * Bug Fixes
     * Fixed bug where a redefined `ansi.style_error` was not being used in all `cmd2` files
 * Enhancements
     * Enabled line buffering when redirecting output to a file
+    * Added `ljustify_text()`, `center_text()`, and `rjustify_text()` to utils.py. All 3 of these functions support
```
[the remark this answers is not in the document]
Comment (kmvanbrunt) [reply CHANGELOG.md]: I'll go with

- `align_left`
- `align_center`
- `align_right`

File: cmd2/utils.py (hunk 1 above)
Comment (kmvanbrunt) [reply cmd2/utils.py]: I'll make it positional

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,11 @@
-## 0.9.22 (TBD, 2019)
+## 0.9.22 (December 9, 2019)
 * Bug Fixes
     * Fixed bug where a redefined `ansi.style_error` was not being used in all `cmd2` files
 * Enhancements
     * Enabled line buffering when redirecting output to a file
-    
+    * Added `align_left()`, `align_center()`, and `align_right()` to utils.py. All 3 of these functions support
```
Comment (tleonhardt) [inline CHANGELOG.md]: It all looks good now

File: docs/api/utility_functions.rst [hunk 4]
```diff
@@ -9,7 +9,13 @@ Utility Functions
 
 .. autofunction:: cmd2.decorators.categorize
 
-.. autofunction:: cmd2.utils.center_text
+.. autofunction:: cmd2.utils.align_text
```
Comment (tleonhardt) [inline docs/api/utility_functions.rst]: Thanks for fixing the Sphinx docs build

**Pull request merged**